Kernel/include/semaphores.h

```diff
@@ -0,0 +1,74 @@
+#ifndef _SEMAPHORE_H_
+#define _SEMAPHORE_H_
+
+#include <stddef.h>
+#include <kernelTypes.h>
+
+
+#define MAX_SEMAPHORES 127
+#define SEM_SUCCES 0
+#define SEM_FAILED -1
+#define SEM_NOTEXISTS -2
+
+typedef int8_t TLock;
+
+/**
+ * @brief Initialize the semaphore system.
+ * 
+ * @returns 0 if the operation succeeded, SEM_FAILED otherwise.
+ *
+ */
+int sem_init();
+
+
+/**
+ * @brief Creates a semaphore with the indicated name, or opens it if
+ * it already existed. 
+ * 
+ * @return The semaphore, or SEM_FAILED (-1) if the operation failed.
```

Comment:
Hey @camilaDiToro. There are some prototypes where the return value between parentesis is not indicated. Either we put them all or we remove them all, it's the same (check sem_init)